keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -829,6 +825,40 @@ configuration.
 | ---- | -------------- | -------- |
 | exempt |            0 | 50%      |
 
+From release 1.32 onward, borrowing is weighted and the weight is
+given in the configuration objects. Prior to release 1.32, borrowing
+is unweighted. Put another way, prior to release 1.32 every priority
+level has the same weight and it is not a field in the configuration
+objects but rather fixed in the implementation.
+
+In release 1.32 the following field is added to
+`LimitedPriorityLevelConfiguration`.
+
+```go
+type LimitedPriorityLevelConfiguration struct {
+  ...
+  // `weight` gives this priority level more or less priority in the borrowing
+  // of concurrency.
+  // The default configuration objects use values in the range 10--100.
+  // The default value is 30.
+  // +optional
+  Weight *int32
+}
+```
+
+Following are the weights in the default non-exempt
+PriorityLevelConfiguration objects.
+
+| Name | Weight |
+| ---- | -----: |
+| leader-election | 100 |
```

Comment:
I feel LE should be higher than node-high & system as it can cause severe disruptions than the other two but not sure what practical difference it'd make with different weight values.